scss/components/_activity-big.scss

```diff
@@ -0,0 +1,123 @@
+.civicase__activity-card.civicase__activity-card--big {
```

Comment:
Why do we need to mention `.civicase__activity-card` here? These two classes are anyhow meant to be used at the same time. Here we just mention the overrides. So I dont see any reason to mention `.civicase__activity-card` here.